src/test/java/org/jenkinsci/plugins/workflow/cps/steps/RestartingLoadStepTest.java

```diff
@@ -157,4 +159,55 @@ public void pauseInsideLoad() throws Exception {
         });
     }
 
+    @Issue("JENKINS-50172")
+    @Test public void loadAndUnnamedClassesInPackage() {
+        story.addStep(new Statement() {
+            @Override public void evaluate() throws Throwable {
+                WorkflowJob p = story.j.jenkins.createProject(WorkflowJob.class, "p");
+                FilePath pkgDir = story.j.jenkins.getWorkspaceFor(p).child("src/org/foo/devops");
+                pkgDir.mkdirs();
+                pkgDir.child("Utility.groovy").write("package org.foo.devops\n" +
+                        "def isValueExist(String value) {\n" +
+                        "    if(value == null || value.trim().length() == 0 || value.trim().equals(\"\\\"\\\"\")) {\n" +
+                        "        return false\n" +
+                        "    }\n" +
+                        "    return true\n" +
+                        "}\n" +
+                        "return this;\n", null);
+                pkgDir.child("JenkinsEnvironment.groovy").write("package org.foo.devops\n" +
+                        "def loadProdConfiguration() {\n" +
+                        "    def valueMap = [:]\n" +
+                        "    valueMap.put('key','value')\n" +
+                        "    return valueMap\n" +
+                        "}\n" +
+                        "return this;\n", null);
+
+                p.setDefinition(new CpsFlowDefinition("def util\n" +
+                        "def config\n" +
+                        "def util2\n" +
+                        "node('master') {\n" +
+                        "    config = load 'src/org/foo/devops/JenkinsEnvironment.groovy'\n" +
+                        "    util = load 'src/org/foo/devops/Utility.groovy'\n" +
+                        "    config.loadProdConfiguration()\n" +
+                        "}\n" +
+                        "util.isValueExist(\"\")\n" +
+                        "semaphore 'wait'\n" +
+                        "node('master') {\n" +
+                        "    util2 = load 'src/org/foo/devops/Utility.groovy'\n" +
+                        "    util = load 'src/org/foo/devops/Utility.groovy'\n" +
```

Comment:
You do not seem to actually be calling any of these utilities to make sure they can run…?